backend/enhanced_spiritual_guidance_router.py

```
@@ -843,5 +843,169 @@ async def get_pricing_dashboard():
+
+@enhanced_router.post("/personalized-remedies", response_model=PersonalizedRemediesResponse)
+async def get_personalized_remedies(
+    request: PersonalizedRemediesRequest,
+    background_tasks: BackgroundTasks
+):
+    """
+    Generate personalized spiritual remedies based on birth details and current issues
+    """
+    try:
+        # Extract birth details
+        birth_details = request.birth_details
```

Ruff (0.12.2)

Local variable `birth_details` is assigned to but never used

Remove assignment to unused variable `birth_details`

(F841)